test/integration/CommunityEndpoints.test.js

```diff
@@ -0,0 +1,246 @@
+import assert from 'assert'
+
+import { Contract, providers, utils, Wallet } from 'ethers'
+
+import StreamrClient from '../../src'
+import * as Token from '../../contracts/TestToken.json'
+
+import config from './config'
+
+function sleep(ms) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms)
+    })
+}
```

Comment:
perhaps https://github.com/streamr-dev/streamr-test-utils#wait

Reply:
Did the change but... is it just me or doesn't `await sleep(1000)` look more readable than `await wait(1000)`... even `await milliseconds(1000)` would be better IMO